src/test/java/org/fundacionjala/trello/runner/Runner.java

```diff
@@ -39,5 +41,6 @@ public void beforeAllScenarios() {
     @AfterTest
     public void afterAllScenarios() {
         // Code executed after features execution.
+        getDriver().quit();
```

Comment:
I get a `NullPointerException` error in this line, when I run all tests by the `Runner` Class

Reply:
yes, you right.. Its solve now